tests/algorithms/test_alibi.py

```diff
@@ -25,13 +22,14 @@ def _double_batch_sequence_length(batch):
     return batch
 
 
-def check_position_embeddings(family, model, max_sequence_length):
-    if family == 'gpt2':
+def check_position_embeddings(model, max_sequence_length):
+    transformers = pytest.importorskip('transformers')
+    if isinstance(model.config, transformers.GPT2Config):
         position_embedding_attribute = 'model.transformer.wpe'
-    elif family == 'bert':
+    elif isinstance(model.config, transformers.BertConfig):
         position_embedding_attribute = 'model.bert.embeddings.position_embeddings'
     else:
-        raise NotImplementedError('Tests not implemented for synthetic_state_family=' + family)
+        raise NotImplementedError('Tests not implemented for synthetic_state_family=' + str(type(model.config)))
```

Comment:
Since we no longer have any synthetic state families, maybe rename to "dummy model family"?

Reply:
yup, makes sense. will do